pyproject.toml

```diff
@@ -49,6 +49,7 @@ toml = "^0.10.2"
 pytest-asyncio = "^0.20.1"
 black = "^22.10.0"
 ruff = "^0.0.244"
+watchdog = "^2.3.1"
```

Comment:
Will we need this in the main dependencies since we need it to be installed in `pip install pynecone`